charts/quickwit/templates/searcher-statefulset.yaml

```diff
@@ -55,6 +55,15 @@ spec:
             - name: "{{ $key }}"
               value: "{{ $value }}"
             {{- end }}
+          {{- if or (.Values.environmentFrom) (.Values.searcher.extraEnvFrom) }}
+          envFrom:
```

Comment:
See this comment: https://github.com/quickwit-oss/helm-charts/pull/85#issuecomment-2176463986